doc/releases/release-notes-3.6.rst

```diff
@@ -246,6 +246,8 @@ Drivers and Sensors
   * ``spi_nor`` driver now sleeps between polls in ``spi_nor_wait_until_ready``. If this is not
     desired (For example due to ROM constraints in a bootloader),
     :kconfig:option:`CONFIG_SPI_NOR_SLEEP_WHILE_WAITING_UNTIL_READY` can be disabled.
+  * ``nrf_qspi_nor`` driver now supports user-configurable QSPI timeout with
```

Comment:
Missing new line before the bullet? Also, please use a devicetree compatible reference.

Reply:
changed to reflect. some other sections do not have a new line but I've added one